py/mpconfig.h

```diff
@@ -490,6 +490,11 @@
 #define MICROPY_LONGINT_IMPL (MICROPY_LONGINT_IMPL_NONE)
 #endif
 
+// Support for calls to buildin.pow() with 3 integer arguments
+#ifndef MICROPY_INT_POW3
+#define MICROPY_INT_POW3 (1)
```

Comment:
Arguably, we'd still want this as 0, as it very niche feature. That maintainers (or community) of individual ports may enable it.